kubectl_deploy/django/deployment.yaml

```diff
@@ -8,27 +8,24 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      app: django-reference
+      app: django-ref-app
   template:
     metadata:
       labels:
-        app: django-reference
+        app: django-ref-app
     spec:
       containers:
         - name: django
           image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/cloud-platform-demo-app:latest
```

Comment:
Best practice is to use a specific tag for a Docker image rather than `latest`, as there's no way of knowing what `latest` points to.

But, if in this case we really do always want to refer to the latest Docker version regardless of what it may be, we should also specify [`imagePullPolicy: Always`](https://kubernetes.io/docs/concepts/containers/images/#updating-images) in the container spec; otherwise Kubernetes won't know that it needs to always check for changes, so the cluster will end up caching the version of this container.